gmtsar/csh/correct_insar_with_gnss.csh

```diff
@@ -112,10 +112,13 @@ errormessage:
 #
   gmt grdfilter tmp1.grd -Dp -Fg17 -Gtmp2.grd
 #
-# upsample to the original size
+# upsample to the original size using gmt surface
+# gmt grd2xyz tmp2.grd | gmt surface -R$insar -T0.5 -Gcorrection.grd
```

Comment:
Should this be -T0.1 to ensure smoothness?

Reply:
Good point, thanks for reminding me! I just updated it in both places to be -T0.1 .